app/services/analytics_events.rb

```diff
@@ -173,6 +173,22 @@ def backup_code_created(enabled_mfa_methods_count:, **extra)
     )
   end
 
+  # Track user creating new BackupCodeSetupForm, record form submission Hash
+  def backup_code_setup_visit(
```

Comment:
Can you add the YARD param documentation for these? Running `make lint_analytics_events` should hopefully highlight, then we add things like
```suggestion
  # @param [Boolean] success
  # @param [Hash] errors
  # @param [Hash] error_details
  def backup_code_setup_visit(
```